### DISCUSSION TITLE
Aplicação do Material Design para uma aplicação

### DISCUSSION THREAD

**Discussion by brunopeople** [frontendbr/forum#1629] [Discussão]
Olá!! 

estou com um problema bem sério e preciso de ajuda de pessoas experiente na parte de front-end, eu estou tentant ouma vaga de emprego e no teste eles requerem que eu faço um pag front-end utilizando o Material Design tanto para desk para mobile, eu nunca sequer utilizei esta ferramenta de framework, como eu não sou muito bem em Design eu não consigo fazer um layout no olho, então qual a ferramente ideial para um desenvolvedor que não é muito bom em design fazer um layout do zero e depois transformar este layout em html e css? Será só existe um Photoshop, não tem um site ou app mais leve com esta função, não é um layout complexo é só

**Comment by rafaelangical** [comment]
Documentação, artigos, videos no youtube ou outro lugar, entender conceitos, etc, se não sabe css recomendo aprender, pois, é essencial.

**Comment by willycamargo** [comment]
@brunopeople referente a esse assunto acredito que dificilmente conseguirá alguma ajuda mais útil do que existe disponível na web.

Da para perceber, pelos termos que usa e a forma como escreve, que você ainda está começando. Eu não gostaria de te desencorajar, mas se implementar um layout com Material Design é um requisito para essa vaga que está tentando evocê não consegue fazer isso sem ajuda, talvez essa vaga não seja para você nesse momento.

Podemos indicar cursos, materiais (pagos ou gratuitos) e até te ajudar com problemas mais pontuais, mas antes você precisa aprender conceitos.

Minhas colocações para esse momento seriam:

1. Sobre ferramentas, não existe apenas Photoshop (apesar dele ser muito usado ainda). Recomendo essas: Sketch (Mac OS), Figma ou Adobe XD. Mas todas elas terão uma curva de aprendizado, será necessário ter paciência pra aprender.


2. Sobre "não ser muito bom em design": eu entendo o que quis dizer aqui, e minha recomendação seria você tentar praticar mais com diferentes layouts. A repetição irá te ajudar a criar referências de implementações que te tornarão cada vez mais fluente no assunto.

**Comment by lucasinocente** [comment]
@brunopeople tu sabe usar outros frameworks / libs como o Bootstrap ou Bulma?

Pra mim o Material é mais um guideline da Google do que um framework. Existem algumas iniciativas que replicam num formato mais simples:

- https://materializecss.com/
- https://getmdl.io/

Mas tu pode tentar também pelo site oficial: https://material.io/develop/web/

**Comment by portothree** [comment]
Como foi citado pelo Lucas é uma guideline, mas como vi que já perguntou aqui no fórum e disse usar Vue, talvez o https://vuematerial.io possa ser útil.
https://github.com/vuematerial/vue-material
